docs/analytics.md

```diff
@@ -0,0 +1,4 @@
+# Analytics
```

Comment:
These files are generated via a script?
If not it would be nice to have a script to do it (in case of new files in the targeted folder)